ymir/command/mir/tools/exporter.py

```diff
@@ -101,8 +101,8 @@ def replace_index_content_inplace(filename: str,
 
             contents = line.strip().split("\t")
             contents[0] = contents[0].replace(asset_search, asset_replace)
-            contents[1] = contents[0].replace(anno_search, anno_replace)
-            line = "\t".join(contents)
+            contents[1] = contents[1].replace(anno_search, anno_replace)
+            print("\t".join(contents), end="\n")
```

Comment:
need clean up?

Reply:
这里的print是print到文件里

Reply:
Optional in-place filtering: if the keyword argument inplace=1 is passed to fileinput.input() or to the FileInput constructor, the file is moved to a backup file and standard output is directed to the input file (if a file of the same name as the backup file already exists, it will be replaced silently).